SpatialGDK/Source/SpatialGDKServices/Private/SpatialCommandUtils.cpp

```diff
@@ -496,4 +497,55 @@ bool SpatialCommandUtils::GetProcessInfoFromPort(int32 Port, FString& OutPid, FS
 	return false;
 }
 
+void SpatialCommandUtils::FetchRuntimeBinary(const FString& RuntimeVersion)
+{
+	FString RuntimePath = FPaths::Combine(SpatialGDKServicesConstants::GDKProgramPath, TEXT("runtime"), RuntimeVersion);
+	SpatialCommandUtils::FetchPackageBinary(RuntimeVersion, SpatialGDKServicesConstants::RuntimeExe, "runtime", RuntimePath, true);
+}
+
+void SpatialCommandUtils::FetchInspectorBinary(const FString& InspectorVersion)
+{
+	FString InspectorPath = FPaths::Combine(SpatialGDKServicesConstants::GDKProgramPath, TEXT("inspector"), InspectorVersion, SpatialGDKServicesConstants::InspectorExe);
+	SpatialCommandUtils::FetchPackageBinary(InspectorVersion, SpatialGDKServicesConstants::InspectorExe, "inspector", InspectorPath, false);
+}
+
+void SpatialCommandUtils::FetchPackageBinary(const FString& PackageVersion, const FString& PackageExe, const FString& PackageName, const FString& SaveLocation, const bool bUnzip)
```

Comment:
This should be returning a bool based on the success of fetching. 
Return true when the package already exists or if we fetched it successfully. Return false otherwise. Same for FetchInspectorBinary and FetchRuntimeBinary